### DISCUSSION TITLE
Filament Spatie Translatable

### DISCUSSION THREAD

**Discussion by fady-ibrahim0** [filamentphp/filament#7320] [General]
I have followed the documentation of filament resources translation and I keep receiving an error when I try to switch between two languages in the ("create page") which says

<img width="1342" alt="Screenshot 2023-07-31 at 9 52 32 PM" src="https://github.com/filamentphp/filament/assets/133704035/60786dc8-8c8c-4e72-b57d-a6baed506cfe">

 So can anyone help?

**Comment by danharrin** [comment]
Missing traits on the pages probably, check the docs
